sdcm/nemesis.py

```diff
@@ -1590,6 +1590,13 @@ def build_list_of_disruptions_to_execute(self, nemesis_selector=None, nemesis_mu
         self.log.debug("This is the list of callable disruptions {}".format(self.disruptions_list))
         return self.disruptions_list
 
+    @property
+    def nemesis_selector_list(self) -> list:
+        nemesis_selector = self.cluster.params.get('nemesis_selector') or []
```

Comment:
I cannot find anything like `nemesis_selector` in SCT's codebase. Maybe we could remove it?
@fruch do you know something about it?